bin/chel

```diff
@@ -1 +0,0 @@
-placeholder, will be replaced by real binary
```

Comment:
@snowteamer Just a heads up, since I missed this in my review. Removing this file was a mistake as it breaks the install.